src/sidebar/store/modules/selection.js

```diff
@@ -51,20 +45,26 @@ TAB_SORTKEYS_AVAILABLE[uiConstants.TAB_ORPHANS] = [
   'Location',
 ];
 
+/**
+ * Utility function that returns all of the properties of an object whose
+ * value is `true`.
+ *
+ * @param {Object} obj
+ * @return {any[]}
```

Comment:
```suggestion
 * @return {string[]}
```

All keys are strings, so you can be more specific here.